grpc_server.go

```diff
@@ -73,6 +78,21 @@ func (s *GRPCServer) Init() error {
 	if s.TLS != nil {
 		opts = append(opts, grpc.Creds(credentials.NewTLS(s.TLS)))
 	}
+
+	if s.PanicHandler != nil {
```

Comment:
It seems to me like the go-grpc-middleware recovery interceptor still provides the ability to return these panics as errors even if you don't want to provide a recovery handler function, so this nil check may not be necessary if I am understanding this correctly: https://github.com/grpc-ecosystem/go-grpc-middleware/blob/main/interceptors/recovery/interceptors.go#L52

What do you think?

Reply:
If we don't do nil check here, and don't provide a `PanicHandler`, the behavior changes from panicing (the current behavior), to returning an error. Would this be a breaking change?